Dockerfile

```diff
@@ -16,12 +16,22 @@ LABEL "maintainer"="https://github.com/laminas/technical-steering-committee/"
 WORKDIR /app
 
 RUN apt update \
+    && apt install -y software-properties-common \
```

Comment:
There might be a way to install the ondrej PPA without using `add-apt-repository`; this was just the easiest way I could figure out.

Reply:
Yeah, seems to be the standard way.

Can you please though put it in a single layer with the next `RUN`? Otherwise, we get two layers moving around a lot of `apt` stuff, potentially unnecessarily